src/core/api/clock.ts

```diff
@@ -292,11 +295,15 @@ function getStalledStatus(
   if (shouldUnstall === true) {
     return null;
   } else if (shouldStall === true || prevStalled !== null) {
-    let reason : "seeking" | "not-ready" | "buffering";
+    let reason : "seeking" | "not-ready" | "buffering" | "internal-seek";
     if (currentState === "seeking" ||
         currentTimings.seeking ||
         prevStalled !== null && prevStalled.reason === "seeking") {
-      reason = "seeking";
+          if (internalSeek) {
+            reason = "internal-seek";
+          } else {
+            reason = "seeking";
+          }
```

Comment:
There's too much indentation here

Reply:
What would be the indentation here? because the statement is inside an `If`:

```ts
 if (currentState === "seeking" ||
        currentTimings.seeking ||
        prevStalled !== null && prevStalled.reason === "seeking") {
          if (internalSeek) {
            reason = "internal-seek";
          } else {
            reason = "seeking";
          }
    } else if (readyState === 1) {
      reason = "not-ready";
    } else {
      reason = "buffering";
    }
```